libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -145,6 +145,19 @@ const AP_Param::GroupInfo AP_AdvancedFailsafe::var_info[] = {
     // @Units: s
     AP_GROUPINFO("RC_FAIL_TIME",   19, AP_AdvancedFailsafe, _rc_fail_time_seconds,    0),
 
+    // @Param: TERM_DLY
+    // @DisplayName: Termination Delay
+    // @Description: This is the time in seconds the aircraft will stay at a failsafe waypoint before aero termination.
+    // @User: Advanced
+    // @Units: seconds 
+    AP_GROUPINFO("TERM_DLY",   20, AP_AdvancedFailsafe, _terminal_delay,   30),
+
+    // @Param: TERM_ENABLE
+    // @DisplayName: Termination Delay Enable
+    // @Description: Enable Terminal Delay                                  
+    // @User Advanced
+    AP_GROUPINFO("TERM_ENABLE",   21, AP_AdvancedFailsafe, _terminate_enable,   1), 
```

Comment:
You don't need this parameter, it overlaps semantically quite badly with the `AFS_ENABLE` and `AFS_TERMINATE` and can be achieved by just using a value of 0 (or less) for the delay as the no delay value.